composer.json

```diff
@@ -42,14 +42,16 @@
     "typo3-ter/seo-basics": "0.9.2",
     "typo3-ter/solr": "5.1.0",
     "pixelant/pxa-form-enhancement": "1.0.1",
-    "typo3-ter/static-info-tables": "^6.3"
+    "typo3-ter/static-info-tables": "^6.3",
+    "typo3-ter/typo3-console": "4.1.2"
   },
   "require-dev": {
     "t3kit/t3kit-docker": "dev-master",
     "t3kit/theme-t3kit": "dev-master",
     "t3kit/theme-t3kit-bluemountain": "dev-master",
     "pixelant/pxa-form-enhancement": "^1.0",
-    "pixelant/pxa-newsletter-subscription": "dev-master"
+    "pixelant/pxa-newsletter-subscription": "dev-master",
+    "lauri/dockert3kit": "~3.0.1"
```

Comment:
@manmath please remove for now this addition will be handled with another ticket.